src/App.js

```diff
@@ -139,12 +141,23 @@ function App() {
     setAppState(newAppState);
   };
 
+  const addNewCardHandler = () => {
+    const newAppState = appState.slice(0);
+    newAppState.unshift({
+      id: uuidv4(),
+      caption: "",
+      text: "",
+      checked: false,
+    });
+    setAppState(newAppState);
+  };
+
```

Comment:
Could you please add a card, that is initially in edit mode? (or even a modal window with card creation, but not necessary).
"Card can be created empty" requirement means that on card initial save, there could be no validation.